mappings/briefs-digital-outcomes-and-specialists-2.json

```diff
@@ -28,10 +28,10 @@
     "briefs": {
       "_meta": {
         "_": "DO NOT UPDATE BY HAND",
-        "version": "10.2.1",
+        "version": "10.3.0",
         "generated_from_framework": "digital-outcomes-and-specialists-2",
```

Comment:
Remind me - this mapping covers both DOS1 and DOS2, right?

Reply:
Yes.